mintapi/api.py

```diff
@@ -407,18 +407,24 @@ def sign_in(
         # mfa screen
         try:
             if mfa_method == "soft-token":
-                mfa_token_input = driver.find_element_by_id("ius-mfa-soft-token")
+                mfa_token_input = driver.find_element_by_css_selector(
+                    "#iux-mfa-soft-token-verification-code, #ius-mfa-soft-token"
+                )
                 if mfa_input_callback is not None:
                     mfa_code = mfa_input_callback(
                         "Please enter your 6-digit MFA code: "
                     )
                 else:
                     mfa_code = oathtool.generate_otp(mfa_token)
                 mfa_token_input.send_keys(mfa_code)
-                mfa_token_submit = driver.find_element_by_id(
-                    "ius-mfa-soft-token-submit-btn"
-                )
+                try:
+                    mfa_token_submit = driver.find_element_by_css_selector(
+                        '#ius-mfa-soft-token-submit-btn, [data-testid="VerifySoftTokenSubmitButton"]'
+                    )
+                except NoSuchElementException:
+                    pass  # no option to select mfa token
                 mfa_token_submit.click()
```

Comment:
How would we click this if there's an exception? Does this need to be inside the try? And should we be passing if there's no element, or is that a sign something went wrong and we need to bail and display a failure message?

Reply:
@mrooney good call. I moved the click inside the try block. But as far as erroring, we'd need to build that logic in for this whole MFA section. Right now it'll pass if there's an error if I'm reading lines 491 and 492 correctly:

```python
except NoSuchElementException:
pass  # not on mfa screen
```

Reply:
Yeah, I guess that was part of my question. If an element wasn't found it would error on trying to click it, and now this changes it to swallow that error more silently. It's fine if there's a good reason I'm not understanding, but if not, we probably shouldn't change the behavior, right?

PS - thanks for this PR and looking forward to merging it!

Reply:
Hey @mrooney, I'm not confident in my error-handling Python abilities. I'd welcome collaboration here or addressing this error handling in a subsequent PR. I just don't think I'm your guy to write in the right error handling.

Reply:
That's fair, I was just asking why error handling is being *added* in this PR that didn't exist before. So in this case I'd prefer it be removed, not added :) Do you mind just removing the try/except block that was added around lines 420, so the diff is just adding the new selectors, and I'll merge it in? Thanks again for your work!

Reply:
@mrooney ah, I misunderstood. This is done!

Reply:
Awesome thanks! 🥳